commit b66d1a442da2a3a6bdad4bcb018acc795b99a165 (refs/changes/21/2221/1)
Author: Trent Mick <trentm@gmail.com>
Date:   2017-07-14T11:32:39-07:00 (2 years, 3 months ago)
    
    joyent/node-docker-registry-client#21 note use of Gerrit and contrib guidelines at top of README

diff --git a/README.md b/README.md
index 5964785..5eaac20 100644
--- a/README.md
+++ b/README.md
@@ -7,6 +7,12 @@ Limitations: Currently only support for Registry API v1
 (i.e. excluding API endpoints for push) and Registry API v2 *pull* support.
 Support for v2 push endpoints is coming.
 
+Note: This repository is part of the Joyent Triton project. See the
+[contribution
+guidelines](https://github.com/joyent/triton/blob/master/CONTRIBUTING.md) --
+*Triton does not use GitHub PRs* -- and general documentation at the main
+[Triton project](https://github.com/joyent/triton) page.
+
 
 ## Install
 
